### PR TITLE
Disable unicode print warning

### LINKED ISSUE #9081: Print warning after importing
```
In [1]: from sympy import *
sympy/printing/pretty/pretty_symbology.py:58: UserWarning: No 'LATIN SUBSCRIPT SMALL LETTER H' in unicodedata
No 'LATIN SUBSCRIPT SMALL LETTER K' in unicodedata
No 'LATIN SUBSCRIPT SMALL LETTER L' in unicodedata
No 'LATIN SUBSCRIPT SMALL LETTER M' in unicodedata
No 'LATIN SUBSCRIPT SMALL LETTER N' in unicodedata
No 'LATIN SUBSCRIPT SMALL LETTER P' in unicodedata
No 'LATIN SUBSCRIPT SMALL LETTER S' in unicodedata
No 'LATIN SUBSCRIPT SMALL LETTER T' in unicodedata

  warnings.warn(unicode_warnings)

In [2]: 
```

This one came with this https://github.com/sympy/sympy/pull/9080

### PR DESCRIPTION
**Pull request by debugger22**
Fixes #9081

~~I've just commented out the lines which were raising the warnings.~~

### CODE REVIEW

**Comment by debugger22** [comment]
@smichr It works on my system. @toolforger could you please confirm.

**Comment by smichr** [comment]
What about this instead,

```
How about this:

```diff
diff --git a/sympy/printing/pretty/pretty_symbology.py b/sympy/printing/pretty/p
index 42cf37c..b9c1f67 100644
--- a/sympy/printing/pretty/pretty_symbology.py
+++ b/sympy/printing/pretty/pretty_symbology.py
@@ -54,9 +54,21 @@ def pretty_use_unicode(flag=None):
         return _use_unicode

     if flag and unicode_warnings:
-        # print warnings (if any) on first unicode usage
-        warnings.warn(unicode_warnings)
-        unicode_warnings = ''
+        # we know that some letters are not supported in Python 2.X so
+        # ignore those warnings:
+        lines = unicode_warnings.splitlines()
+        if lines:
+            known = ['LATIN SUBSCRIPT SMALL LETTER %s' % i for i in 'HKLMNPST']
+            keep = []
+            for l in lines:
+                if any(i in l for i in known):
+                    continue
+                keep.append(l)
+        if keep:
+            unicode_warnings = '\n'.join(keep)
+            # print warnings (if any) on first unicode usage
+            warnings.warn(unicode_warnings)
+            unicode_warnings = ''

     use_unicode_prev = _use_unicode
     _use_unicode = flag
```

**Comment by debugger22** [comment]
Works on my system. +1
I've updated the PR and changed the authorship.

File: sympy/printing/pretty/pretty_symbology.py [hunk 1]
```diff
@@ -54,9 +54,21 @@ def pretty_use_unicode(flag=None):
         return _use_unicode
 
     if flag and unicode_warnings:
-        # print warnings (if any) on first unicode usage
```
Comment (toolforger) [inline sympy/printing/pretty/pretty_symbology.py]: This should be left in I think.

**Comment by toolforger** [comment]
Confirming that current patch does not emit any Unicode warnings.

**Comment by smichr** [comment]
Sorry @debugger22 ... what about #9084 which refactors code a bit so it is easier to update when 2.X is dropped. This also fixes a latex issue.

**Comment by debugger22** [comment]
It's ok. :)
Let's close this one.

**Pull request closed without merging**